### PR TITLE
Fix ssm parameter naming convention

### PR DESCRIPTION
**Pull request by armins88**

### CODE REVIEW

File: src/services/ssm.service.ts [hunk 1]
```diff
@@ -12,8 +12,12 @@ export interface Parameters {
 export type GetParameters = () => Reader<AWS.SSM, Promise<Parameters>>
 
 export const getParameters: GetParameters = () => async (ssmClient) => {
-  // TODO Change ssmParameterStoreName for your project
-  const ssmParameterStoreName = '/default/node_example_lambda'
+  // We need to pull one env var from environ. The rest should come from ssm
+  // eslint-disable-next-line no-process-env
+  const nodeEnv = process.env.nodeEnv
```
Comment (amsul) [inline src/services/ssm.service.ts:17]: 🤔 isn't this env named `NODE_ENV`?

Comment (armins88) [reply src/services/ssm.service.ts]: It was a typo. 
fixed it
thanks

File: src/lambda.ts [hunk 2]
```diff
@@ -11,5 +11,6 @@ export async function handler(
   // Lambda entry
   logger.info(`Received event: ${event.httpMethod.toUpperCase()} ${event.path}`)
 
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
```
Comment (amsul) [inline src/lambda.ts:14]: Oh I'm surprised this was needed even after the `as APIGatewayProxyResult` was added 🤔

File: src/services/ssm.service.ts [hunk 3]
```diff
@@ -12,8 +12,12 @@ export interface Parameters {
 export type GetParameters = () => Reader<AWS.SSM, Promise<Parameters>>
 
 export const getParameters: GetParameters = () => async (ssmClient) => {
-  // TODO Change ssmParameterStoreName for your project
-  const ssmParameterStoreName = '/default/node_example_lambda'
+  // We need to pull one env var from environ. The rest should come from ssm
+  // eslint-disable-next-line no-process-env
+  const nodeEnv = process.env.NODE_ENV
+  const ssmParameterStoreName = `/${
+    nodeEnv != null ? nodeEnv : 'dev'
```
Comment (amsul) [inline src/services/ssm.service.ts:19]: Fallback value for `NODE_ENV` should be `development`

```suggestion
    nodeEnv != null ? nodeEnv : 'development'
```

File: src/services/ssm.service.ts [hunk 4]
```diff
@@ -12,8 +12,12 @@ export interface Parameters {
...
+  }/user-events`
```
Comment (amsul) [inline src/services/ssm.service.ts:20]: Is this user-events part intentional?

File: src/services/ssm.service.ts (hunk 3 above)
Comment (armins88) [reply src/services/ssm.service.ts:19]: `dev` is the convention infra-legacy uses for all its environments.

File: src/services/ssm.service.ts (hunk 4 above)
Comment (armins88) [reply src/services/ssm.service.ts]: Oof copy pasta

Comment (armins88) [reply src/services/ssm.service.ts]: Fixed

**Review by logansease — APPROVED** [review]

File: src/services/ssm.service.ts (hunk 3 above)
Comment (amsul) [reply src/services/ssm.service.ts:19]: Are we sure that's NODE_ENV and not APP_ENV?

**Pull request merged**